src/Rules.Framework/PriorityCriterias.cs

```diff
@@ -8,11 +8,11 @@ public enum PriorityCriterias
         /// <summary>
         /// Sets the rule with the lowest priority number to win on a untie scenario.
         /// </summary>
-        TopmostRuleWins = 0,
+        TopMostRuleWins = 0,
 
         /// <summary>
         /// Sets the rule with the highest priority number to win on a untie scenario.
         /// </summary>
-        BottommostRuleWins = 1
+        BottomMostRuleWins = 1
```

Comment:
This is a breaking change and it is what is causing the build to break: https://github.com/Farfetch/rules-framework/actions/runs/3567499476/jobs/5995271630.

Reply:
Reverted change.